sc/social/like/browser/viewlets.py

```diff
@@ -61,7 +67,16 @@ class SocialMetadataViewlet(BaseLikeViewlet):
     """Viewlet used to insert metadata into page header
     """
     render = ViewPageTemplateFile('templates/metadata.pt')
-    render_method = 'metadata'
+
+    def __init__(self, context, request, view, manager):
```

Comment:
once again, don't add any code into the `__init__()` method or we can get unexpected errors.

Reply:
Done